### ISSUE TITLE
Possible to have a list of what NOT to capture?

### ISSUE THREAD

**Issue by ataylor32** [squareboat/sneaker#36]
Like issue #24, I'd like to capture everything with some exceptions:

- Illuminate\Auth\AuthenticationException
- Illuminate\Database\Eloquent\ModelNotFoundException
- Illuminate\Session\TokenMismatchException
- Symfony\Component\HttpKernel\Exception\NotFoundHttpException

Looking at the `Sneaker` class' `shouldCapture` method, this doesn't appear to be possible right now. Would you be willing and able to add this functionality?

**Comment by akaamitgupta** [comment]
@ataylor32 You can just check the type of exception before calling `app('sneaker')->captureException($exception)` in your code and it will solve your problem.

**Comment by ataylor32** [comment]
@akaamitgupta Thank you. I don't have a lot of Laravel experience. I saw your comment on #24, but didn't know what `$this->shouldReport($exception)` was referring to. I see now that it's from `Illuminate\Foundation\Exceptions\Handler`, which is what `App\Exceptions\Handler` extends. So I changed the `report` method from this:

```php
app('sneaker')->captureException($exception);
```

...to this:

```php
if ($this->shouldReport($exception)) {
    app('sneaker')->captureException($exception);
}
```

I also have Sneaker's `capture` array set to `'*'`. That solved it. Thanks again!
